.changeset/rude-dolphins-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Fixed carousel dot button's incorrect height.
```

Comment:
nit

```suggestion
Fixed `Carousel.Dot`'s incorrect height.
```

Also, in the css changeset, should it be "carousel dot button's" or "carousel dot's"?

Reply:
Committed your suggested change.

> should it be "carousel dot button's" or "carousel dot's"?

I think it should be "carousel dot button's" because it is the button wrapping the dots that needed the fix, not the dots themselves.